### PR TITLE
Add DAMIC (2020), update PandaX-II (2020) limit

### PR DESCRIPTION
**Pull request by cwittweg**
- Added the 2020 DAMIC limit (https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.125.241803).
- Updated the PandaX-II 2020 limit with a new version that Laura obtained from Xun Chen on 26 August 2021. Current file is ten days older.

pandalimits

### CODE REVIEW

**Comment by kdund** [comment]
@revive would you have a chance to have a look? Thanks!

**Review by revive — APPROVED** [review]
Yes. This one for PandaX-II 2020 is the published version.

File: data/DAMIC_2020.csv [hunk 1]
```diff
@@ -0,0 +1,23 @@
+mass,cross_section
```
Comment (kdund) [inline data/DAMIC_2020.csv:1]: could you add a comment-# in front of the headers, I think github then parses it as .csv?

**Review by kdund — CHANGES_REQUESTED** [review]

File: data/DAMIC_2020.csv (hunk 1 above)
Comment (cwittweg) [reply data/DAMIC_2020.csv]: Okay, done. This is not present in the other .csv-files (if they include headers).

Comment (kdund) [reply data/DAMIC_2020.csv]: do as I say etc.... ;)

**Review by kdund — APPROVED** [review]

**Pull request merged**